server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/sql/DropColumnsBuilder.java

```diff
@@ -40,33 +46,45 @@ public DropColumnsBuilder(Dialect dialect, String tableName, String... columns)
     this.columns = columns;
   }
 
-  public String build() {
-    StringBuilder sql = new StringBuilder().append("ALTER TABLE ").append(tableName).append(" ");
+  public List<String> build() {
     switch (dialect.getId()) {
       case PostgreSql.ID:
       case MySql.ID:
-        dropColumns(sql, "DROP COLUMN ");
-        break;
+        StringBuilder sql = new StringBuilder().append("ALTER TABLE ").append(tableName).append(" ");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "ALTER TABLE " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)